src/relation_engine_server/api.py

```diff
@@ -65,13 +65,17 @@ def show_view(name):
     return flask.Response(spec_loader.get_view(name), mimetype='text/plain')
 
 
-@api.route('/refresh_specs', methods=['GET'])
+@api.route('/update_specs', methods=['GET'])
 def refresh_specs():
     """
-    Manually pull from the spec git repo to get updates.
+    Manually check for updates, download spec releases, and init new collections.
     """
-    git_output = pull_spec.pull_spec()
-    return flask.jsonify({"updates": git_output})
+    auth.require_auth_token(['RE_ADMIN'])
```

Comment:
Good call.

Reply:
Although you'll need to add a service token (long lasting) to the cron job that calls this